app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  has_many :job_experiences
```

Comment:
This and the has_many for JobExperiences need to become a has_many :through relationship.